internal/monitors/filesystems/filesystems.go

```diff
@@ -85,14 +84,10 @@ type Monitor struct {
 // returns common dimensions map according to reportInodes configuration
 func (m *Monitor) getCommonDimensions(partition *gopsutil.PartitionStat) map[string]string {
 	dims := map[string]string{
-		"mountpoint":      strings.Replace(partition.Mountpoint, " ", "_", -1),
+		"mountpoint":      strings.Replace(strings.Replace(partition.Mountpoint, m.hostFSPath, "", 1), " ", "_", -1),
```

Comment:
What happens if `m.hostFSPath` is `""`?  Is the inner `strings.Replace` a noop in that case?

Reply:
yeah it is

Reply:
`If old is empty, it matches at the beginning of the string and after each UTF-8 sequence, yielding up to k+1 replacements for a k-rune string.`